src/store/actions/posts.js

```diff
@@ -0,0 +1,3 @@
+export const getAllPosts = (payload) => {
```

Comment:
My bad, forgot to mention this earlier, but actions names are always going to be events. So this one can be something like `postsFetched` and `POSTS_FETCHED`. Action names should not be imperative, they should be declarative. They signify an event in the lifecycle of the app.

Good reading - https://twitter.com/dan_abramov/status/800310397538619393?lang=en

Reply:
ok done.